dump/src/util/dump/Dump.java

```diff
@@ -1186,6 +1201,20 @@ private void renameFile( File currentFile, File newFile ) throws IOException {
       _log.info("renamed file " + currentFile + " to " + newFile);
    }
 
+   private void resetMeta() {
+      _sequence = (long)(Math.random() * 1000000);
+      _metaData.clear();
+   }
+
+   private void setExternalizationVersionInMetaData() {
+      externalizationVersion version = _beanClass.getAnnotation(externalizationVersion.class);
+      if ( version != null ) {
+         _metaData.put("externalizationVersion", "" + version.version());
+      } else {
+         _metaData.remove("externalizationVersion");
+      }
```

Comment:
This should just use `getVersionFromCode`, no?

Reply:
Should be possible to reuse, though not strictly required...

Reply:
Fixed by always writing code version into dump meta file.